handshake_client.go

```diff
@@ -251,6 +251,8 @@ func (hs *clientHandshakeState) handshake() error {
 		hs.keySchedule = newKeySchedule13(hs.suite, c.config, hs.hello.random)
 		hs.keySchedule.write(hs.hello.marshal())
 		hs.keySchedule.write(hs.serverHello.marshal())
+
+		c.scts = hs.serverHello.scts
```

Comment:
I think this should be done in ``func (hs *clientHandshakeState) doTLS13Handshake() error {``. Do you mind moving it?